icu4c/source/test/testdata/numberpermutationtest.txt

```diff
@@ -3504,7 +3504,7 @@ measure-unit/length-furlong unit-width-full-name rounding-mode-floor
 percent unit-width-narrow integer-width/##00
   es-MX
     00 %
-    1827.3645 %
+    1,827.3645 %
```

Comment:
All of the changes in this file are related to CLDR changing the minimumGrouping for es_419 locales from 2 to 1.